runtime/src/identity.rs

```diff
@@ -568,6 +533,56 @@ decl_module! {
                 Err("No did linked to the user")
             }
         }
+
+
+        // Manage Authorizations to join to an Identity
+        // ================================================
+
+        /// The key designated by `origin` accepts the authorization to join to `target_id`
+        /// Identity.
+        ///
+        /// # Errors
+        ///  - Key should be authorized previously to join to that target identity.
+        ///  - Key is not linked to any other identity.
+        pub fn authorize_join_to_identity(origin, target_id: IdentityId) -> Result {
```

Comment:
This function should be added to the list of functions not considered by the `CurrentDid` SigningExtension (so that it can be called by an account which is not yet part of an identity).

Reply:
Fixed.

Reply:
Fixed.